amplipi/streams.py

```diff
@@ -1199,19 +1199,17 @@ def connect(self, src):
       # TODO: allow port to be specified with server (embedding it in the server URL does not work)
 
       self.proc = subprocess.Popen(args=lms_args)
-      self._connect(src)
-
-
+      self._connect(vsrc)
```

Comment:
Please revert this change, connect should not be called directly by a PersistantStream.